app/components/NoticeTypeCheckboxes.tsx

```diff
@@ -157,29 +159,161 @@ const NoticeTypes = {
+      if (userSelected[key]) {
+        selectedTotal++
+      }
+    }
+    setSelectedCounter(selectedTotal)
+
+    if (metrics) {
+      setAlertEstimate(getSum(metrics))
+    }
+  }
+
+  function getSum(metrics: FormattedData) {
+    return metrics[selectedFormat]
+      .filter(
+        (item) =>
+          userSelected[item.topic] !== undefined &&
+          userSelected[item.topic] == true
+      )
+      .map((item) => item.latestRate)
+      .reduce((partialSum, a) => partialSum + a, 0)
+  }
+
+  function getClassName(latestRate: number | undefined): string {
+    if (latestRate == undefined) {
+      return 'pill-undefined'
+    }
+    if (latestRate < 25) {
+      return 'pill-low-rate'
+    } else if (latestRate < 50) {
+      return 'pill-med-rate'
+    } else {
+      return 'pill-high-rate'
+    }
+  }
+
   return (
-    <NestedCheckboxes
-      nodes={Object.entries(NoticeTypes).map(([mission, noticeTypes]) => ({
-        id: mission,
-        label: mission,
-        name: '',
-        nodes: noticeTypes.map((noticeType) => ({
-          id: noticeType,
-          label: noticeType,
-          name: noticeType,
-          className: 'sub-option',
-          defaultChecked: defaultSelected
-            ? defaultSelected.indexOf(noticeType) > -1
-            : false,
-        })),
-      }))}
-    />
+    <>
+      <NestedCheckboxes
+        nodes={Object.entries(NoticeTypes).map(([mission, noticeTypes]) => ({
+          id: mission,
+          label: mission,
+          name: '',
+          nodes: noticeTypes.map((noticeType) => ({
+            id: noticeType,
+            label: (
+              <>
+                {noticeType}
+                {metrics ? (
+                  <div className="padding-left-1 display-inline">
+                    <span
+                      title="Average rate of alerts per day"
+                      className={
+                        'border-2px radius-lg padding-2px ' +
+                        getClassName(
+                          metrics[selectedFormat].find(
+                            (item) => item.topic == noticeType
+                          )?.latestRate
+                        )
+                      }
+                    >
+                      {metrics[selectedFormat].find(
+                        (item) => item.topic == noticeType
+                      )?.latestRate ?? '?'}
+                    </span>
+                  </div>
+                ) : null}
+              </>
+            ),
+            name: noticeType,
+            className: 'sub-option',
+            defaultChecked: defaultSelected
+              ? defaultSelected.indexOf(noticeType) > -1
+              : false,
+          })),
+        }))}
+        childoncheckhandler={counterfunction}
+      />
+      {showTotalSelected ? (
+        <div>
+          {selectedCounter} selected alerts for an estimated total of{' '}
+          {alertEstimate} alerts per day
+        </div>
+      ) : null}
```

Comment:
1. [ ] Display "0 selected alerts for an estimated total of 0 alerts per day" if nothing is selected
2. [ ] Display correct singular/plural form